docs/guides/ide/startide.md

```diff
@@ -4,7 +4,7 @@
 
 Steps for the Merlin administrator:
```

Comment:
@eric-simpson Above this line, the docs link to the [Merlin Overview](https://www.ibm.com/support/pages/node/6574839) support page which also looks to be outdated at least in terms of images. Do we need to update this as well?

Reply:
the link to the merlin overview support page should be removed